spec/support/checkboxes_spec_helper.rb

```diff
@@ -0,0 +1,17 @@
+module CheckboxesSpecHelper
```

Comment:
El `Spec` no me parece necesario. Creo que nace de que esté archivo inicialmente nace como una copia de `ApplicationSystemTestCase`

```suggestion
module CheckboxesHelper
```